README.md

```diff
@@ -54,6 +54,17 @@ Installation in Composer project (method 2)
 * Use the coding standard with `./vendor/bin/phpcs --standard=PHPCompatibility`
 
 
+Installation via a git check-out to an arbitrary directory (method 3)
+-----------------------
+
+* Install [PHP_CodeSniffer](https://github.com/squizlabs/PHP_CodeSniffer) via [your preferred method](https://github.com/squizlabs/PHP_CodeSniffer#installation) (Composer, PEAR, Phar file, Git checkout).
+* Checkout the latest release from https://github.com/wimg/PHPCompatibility/releases into an arbitrary directory.
+* Add the path to the directory **above** the directory in which you cloned the PHPCompability repo to the PHPCS Configuration:
```

Comment:
Thanks for the feedback, I've adjusted the text to have a proper example and included a link to the PHPCS wiki as well.

Please let me know if it's still not clear enough and I'll have another think about the phrasing.